frontend/src/metabase/embedding/components/SharingMenu/MenuItems/PublicLinkMenuItem.tsx

```diff
@@ -22,6 +25,11 @@ export function PublicLinkMenuItem({
         data-testid="embed-menu-public-link-item"
         leftSection={<Icon name="link" aria-hidden />}
         onClick={onClick}
+        {...(!isPublicSharingEnabled && {
+          onClick: undefined,
+          component: "div",
+          className: cx(MenuStyles.ignoreHover, CS.textLight),
```

Comment:
I think we can avoid custom css here

```suggestion
          component: "div",
          disabled: true,
          c: "text-light",
```